pkg/statscombiner/statscombiner_test.go

```diff
@@ -0,0 +1,82 @@
+// Copyright 2022 The Inspektor Gadget authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package statscombiner
+
+import "testing"
+
+func TestStatCombiner(t *testing.T) {
+	sc := NewStatsCombiner[int](2)
+
+	type testRun struct {
+		Name            string
+		IntervalStats   map[string][]int
+		ExpectedEntries int
+	}
+
+	tests := []testRun{
+		{
```

Comment:
Could you please add some additional "risky" tests? For instance, adding this one as first:

```golang
		{
			Name:            "try to get stats from a just initialised combiner",
			ExpectedEntries: 0,
		},
```